wordpress-mobile/WordPress-iOS
# Today Widget: update no connection view

Ref #13271

This updates the no connection view in the _Today_ widget added in #13266. It is displayed when:
- There is no network connection.
- There is no saved data.

If there is saved data, it will be displayed.

To test:

Side note: In the simulator, the widget was never notified when the network connection was re-established. I suggest testing on a real device as the reachability notifications work as expected. 

Since `No network available` will only be displayed if there is no saved data, the data needs to be purged, and then the network disabled.

---
- In the app, go to Stats > Widgets > `Use this site`.
- Add the _Today_ widget to the Today view.
- Return to the app and change the site used for widgets (so the saved data is purged).
- Disable internet connection.
- Return to the Today view.
- Verify the new no connection view is displayed.

![no_network](https://user-images.githubusercontent.com/1816888/73024112-29bb3e00-3dea-11ea-9dea-1a2135182a67.jpeg)

---
- Enable internet connection.
- Verify the data loads.

![loaded](https://user-images.githubusercontent.com/1816888/73018072-17d39e00-3dde-11ea-8629-8fc0c50acf69.jpeg)

---
- Disable internet connection again.
- Verify the data is still displayed.

---
PR submission checklist:
